Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM registry.gitlab.com/pages/hugo/hugo_extended:latest
+FROM klakegg/hugo:0.101.0-ext-alpine
```

Comment:
So, it looks like this is just a random individual's docker container.

While it looks like it was pinned in [Hugo's forum](https://discourse.gohugo.io/t/klakeggs-wonderful-hugo-docker-image-update-thread/20347), is there any reason why you went with that instead of Gitlab's?

Just looking at the forum they don't appear to be a Hugo maintainer and don't have a mod icon next to their name.

Reply:
The image is the one listed on the Hugo project website, that's why. In lieu of an official image it seems prudent to use the closest thing we have to one.

This won't be the final production image I don't think. Only dev / CI. 

So until there's a solid technical reason for one image over the other my view would be to use the only image Hugo themselves promote on their website.

Reply:
> So until there's a solid technical reason for one image over the other my view would be to use the only image Hugo themselves promote on their website.

So, my only qualm with using the other image is from a security perspective, that even though they are promoting it. It's not backed by one of their team member or by something/someone who's got more of a reputation.

One day their project could be perfectly fine, and then the next they could pull a [protestware stunt](https://arstechnica.com/information-technology/2022/03/sabotage-code-added-to-popular-npm-package-wiped-files-in-russia-and-belarus/) the next day and if JB relies on that for their main website deployments it could modify/alter the deployment.

While with the Gitlab image, I'm sure technically it *might* be possible, but there is a significantly lower chance of things happening to it because it's produced by Gitlab employees on Gitlab infrastructure.

Also, the hugo docs do recommend using that image, but it's just on the Gitlab deployment page: https://gohugo.io/hosting-and-deployment/hosting-on-gitlab/

Reply:
The image itself doesn't really matter too much I guess.

From my time at Linuxserver.io I gravitate towards Alpine images. I'll probably be around to fix any image issues that crop up too so I'm not too worried about that aspect. Though I would prefer Hugo to just make an image overall.

Reply:
> Though I would prefer Hugo to just make an image overall.

Ya, I'd definitely prefer this! :sweat_smile: 

Also, the gitlab image is an alpine image. I thought it was, but I couldn't confirm till I got to my desktop (on mobile before).
![image](https://user-images.githubusercontent.com/10230166/183528898-34b777fc-1328-4b5b-a210-bfd297a1f0fc.png)

Reply:
So, based on all that we've talked about I'd recommend just keeping the gitlab image:

```suggestion
FROM registry.gitlab.com/pages/hugo/hugo_extended:latest

```